src/api/transactions.coffee

```diff
@@ -421,7 +448,16 @@ exports.updateTransaction = (transactionId) ->
       if not autoRetryUtils.reachedMaxAttempts tx, channel
         updates.autoRetry = true
         autoRetryUtils.queueForRetry tx
-
+    
+    transactionToUpdate = yield transactions.Transaction.findOne({ _id: transactionId }).exec()
+    bodiesLength = 0
```

Comment:
Maybe rename this to `defaultBodiesLength` just to indicate that this is the default start for the calculate function